docs/workflow operations/Optimize Doc.md

```diff
@@ -0,0 +1,107 @@
+# Optimize:
+## Goal
+Used to find the minimal or maximum value to a given parameter value or time that a QoI (Quality of Interest) constraint does not exceed the provided threshold.
+
+We can ask questions to try to find out a good start time for an intervention:
+
+I want to turn on "masking" (beta set to a smaller value ~0.1) 
```

Comment:
If this is a quote (question being asked) it should have quotes around it, otherwise the mixing of plural and singular pronouns is just confusing.